### PR TITLE
Add Gary!

### PR DESCRIPTION
**Pull request by sampoder**

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝Changed routes:


| [**/team**](https://site-iwpzn6p33.hackclub.dev/team) |
|:-:|
|<a href="https://site-iwpzn6p33.hackclub.dev/team">Screenshot of /team</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/1JFERlSCH5YHYs7sH_SL-U2Lr-htNDiaUwmrYXG2WlTHagwXMR4BxpBqsmD6pWWTIeNZjonFPUk4ZkB2AGeY1g.png">(view full size)</a> |







Commit a19258235744c353ddb576a4ece8b3fcb0dd21c5 (https://site-iwpzn6p33.hackclub.dev).

File: src/pages/team.js [hunk 1]
```diff
@@ -98,6 +98,12 @@ export default () => (
           text="Melanie grew up in northern New England where she obtained a degree in Marine Biology. She then spent several years running a pet store with 20+ employees and recently decided to change career paths. This led her to Hack Club where she is excited about helping students pursue their dreams."
           img="/team/mel.png"
         />
+        <Bio
+          name="Gary Tou"
+          teamRole="Bank Operations"
+          text="Gary lives in Washington and is currently completing his High Schooling. He used Hack Club Bank to form the synHacks organization and help his local community. Now, he is helping make the product (HCB) that enabled him to do great things even greater for others. "
```
Comment (garyhtou) [inline src/pages/team.js]:
```suggestion
          text="Gary is an aspiring developer from Seattle who loves everything tech. After using Hack Club Bank to launch the synHacks organization and inspire students to help the global community, Gary joined Hack Club to make the product that enabled him to do great things even greater for others."
```

File: src/pages/team.js [hunk 2]
```diff
@@ -98,6 +98,12 @@ export default () => (
...
+          img="/team/gary.jpeg"
```
Comment (garyhtou) [inline src/pages/team.js:106]:
```suggestion
          pronouns="he/him"
          img="/team/gary.jpeg"
```

**Review by garyhtou — CHANGES_REQUESTED** [review]
Thanks @sampoder!

**Comment by sampoder** [comment]
Website gary is sad

**Review by maxwofford — APPROVED** [review]
I'm cool with this, but let's get @MelSmith104's approval on this first.

I'd also like the copy to be a bit shorter. The content is fine, but there are some filler words we can trim.

"launch the synHacks organization and inspire students to help the global community" -> "launch synHacks, a social nonprofit," or something like that

File: src/pages/team.js [hunk 3]
```diff
@@ -98,6 +98,13 @@ export default () => (
           text="Melanie grew up in northern New England where she obtained a degree in Marine Biology. She then spent several years running a pet store with 20+ employees and recently decided to change career paths. This led her to Hack Club where she is excited about helping students pursue their dreams."
           img="/team/mel.png"
         />
+        <Bio
+          name="Gary Tou"
+          teamRole="Bank Operations"
+          text="Gary is an aspiring developer from Seattle who loves everything tech. After using Hack Club Bank to launch the synHacks organization and inspire students to help the global community, Gary joined Hack Club to make the product that enabled him to do great things even greater for others."
```
Comment (sampoder) [inline src/pages/team.js:104]:
```suggestion
          text="Gary is a developer from Seattle. After using Hack Club Bank to launch the synHacks organization, a social nonprofit, Gary joined Hack Club to make the product that enabled him to do great things even greater for others."
```

**Pull request merged**